binstar_client/utils/projects/inspectors.py

```diff
@@ -0,0 +1,40 @@
+class ProjectFilesInspector(object):
+    def __init__(self, pfiles):
+        self.pfiles = pfiles
+
+    def update(self, metadata):
+        metadata['files'] = [pfile.to_dict() for pfile in self.pfiles]
+        return metadata
+
+
+class DocumentationInspector(object):
+    valid_names = [
+        'README.md',
+        'README.rst',
+        'README.txt',
+        'README'
+    ]
```

Comment:
same basic comment here, if this is logic of interest only to upload it may as well live here, if it's logic that should be shared elsewhere it should live in anaconda-project. For the project name I think definitely shared; for the file list and readme, not clear to me that it needs to be.

The main "elsewheres" are Anaconda server side, the Qt UI (Navigator), and our command line tool suite.